source/MonoGame.Extended.Input/InputListeners/GamePadListener.cs

```diff
@@ -71,7 +71,7 @@ public GamePadListener(GamePadListenerSettings settings)
             RepeatDelay = settings.RepeatDelay;
 
             _previousGameTime = new GameTime();
-            _previousState = GamePadState.Default;
+            _previousState = new GamePadState();
```

Comment:
GamePadState.Default is an MG extension that's wrapping `new GamePadState()`. It's removed from KNI.

Reply:
Looking in MonoGame, I'm not sure what the intent is for `GamePadState.Default`.  I would think it would be a way of reducing allocations, but it's only implemented in `GamePadState`.  `KeyboardState` doesn't have one and `MouseState` has one that's `private` and only return when getting state when the window isn't in focus. 

It also doesn't seem to be part of the original [XNA Api](https://learn.microsoft.com/en-us/previous-versions/windows/xna/bb197012(v=xnagamestudio.10)).

That's just to document and say change is good. Thanks

Reply:
Originally it was an internal static field, to quickly return a copy, which turned into a public property. 
It doesn't do anything more than `new GamePadState()` or default(GamePadState) would do.
https://github.com/MonoGame/MonoGame/commit/423cef16beb068cd8d01cc659c4fd96c5284c162